packages/desktop-client/src/components/reports/ReportTable.tsx

```diff
@@ -0,0 +1,416 @@
+import React, { useLayoutEffect, useRef, memo } from 'react';
+
+import * as d from 'date-fns';
+//import { useSelector } from 'react-redux';
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏can be removed